apps/sms/views/shared/style/composer.css

```diff
@@ -197,6 +199,15 @@ form[role="search"] button[type="submit"]:after {
   transform: translate(.2rem);
 }
 
+/* 
+ * https://bugzilla.mozilla.org/show_bug.cgi?id=1055357
+ * Margin reset for send button, since role="search" attribute was deleted,
+ * actually this thing broke look
+ */
```

Comment:
I think the comment is not really useful here. A useful comment is a comment that can be useful for a future reader to understand the code.

So what could be useful here? Simply that we're overriding a CSS rule from the building blocks. Can you please just state this ? Something like "Overriding button margin from Building Blocks"